shared/src/commonTest/kotlin/co/touchlab/kampstarter/BaseTest.kt

```diff
@@ -0,0 +1,7 @@
+package co.touchlab.kampstarter
+
+import kotlinx.coroutines.CoroutineScope
+
+expect abstract class BaseTest(){
+    fun <T> runTest(block: suspend CoroutineScope.() -> T)
```

Comment:
This could stay top-level, but it doesn't make a huge difference in this case. I bring it up only because this is the sort of expect/actual I find myself avoiding in production code. An expect class can't be switched out for a test implementation unless you have an interface as well, at which point the expect often doesn't add much. (There's exceptions, like the empty typalias with extensions, but that's usually not the first thing people try)